samples/server/petstore/jaxrs-spec/swagger.json

```diff
@@ -760,24 +760,24 @@
           "description" : "None",
           "required" : true,
           "type" : "number",
-          "maximum" : 543.2,
-          "minimum" : 32.1
+          "maximum" : 543.200000000000045474735088646411895751953125,
+          "minimum" : 32.10000000000000142108547152020037174224853515625
         }, {
           "name" : "float",
           "in" : "formData",
           "description" : "None",
           "required" : false,
           "type" : "number",
-          "maximum" : 987.6,
+          "maximum" : 987.6000000000000227373675443232059478759765625,
```

Comment:
Just to be clear. I'm not saying your PR causes the issue we're seeing.

Looks like we discovered some new issues, probably due to Swagger Parser. I'm actually performing more tests to confirm the "behaviour".